src/assets/css/bootstrap.min.css

```diff
@@ -11280,3 +11283,19 @@ fieldset:disabled .btn {
         display: none !important;
     }
 }
+
+/*  */
+
+.custom-link {
```

Comment:
Why are you making changes in bootstrap.min.css. This file should never be changed?

Reply:
done

Reply:
@karun-rumsan OK. Make the changes and send PR to stage.